src/config.env.js

```diff
@@ -0,0 +1,59 @@
+
+const getMaxWaitTime = () => {
+  const maxWait = +process.env.KAFKA_CONSUMER_MAX_WAIT_TIME_MS;
+  if (maxWait === 0 || maxWait === NaN) {
+    return 100;
+  }
+
+  return maxWait;
+};
+
+const getMaxBytes = () => {
+  const maxBytes = +process.env.KAFKA_CONSUMER_MAX_BYTES;
+  if (maxBytes === 0 || maxBytes === NaN) {
+    return 1024 * 1024;
+  }
+
+  return maxBytes;
+};
+
+const getIdleTimeout = () => {
+  const idleTimeout = +process.env.KAFKA_CONSUMER_IDLE_TIMEOUT;
+  if (idleTimeout === 0 || idleTimeout === NaN) {
+    return 1000;
+  }
+
+  return idleTimeout;
+};
+
+const herokuConfig = {
+  topics: process.env.TOPICS ? process.env.TOPICS.split(',').map((string) => string.trim()) : [''],
```

Comment:
`['']` is an array of one element where the element is an empty string, but it should be `[]` i.e. an empty array, array with zero elements